schema.lua

```diff
@@ -61,6 +61,14 @@ return {
                       type = "string",
                     },
                   },
+
+                  {
+                    -- Example: true / false
+                    allow_if_undefined = {
```

Comment:
```suggestion
                    allow_if_undefined = {
```

Can we rename this to `allow_undefined` please?  I think that's a little clearer/more-concise.